### PR TITLE
Adds PWA stat for the Raphael Hotel.

### PR DESCRIPTION
**Pull request by derekshirk**
## Overview

My first entry. Adds PWA stat for the Raphael Hotel.

## Screenshots

screen shot 2017-07-13 at 3 55 34 pm

---

/CC @cloudfour/pwastats

### CODE REVIEW

File: _posts/2017-07-13-raphael-hotel.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+sourceURL: http://www.prweb.com/releases/2017/07/prweb14501233.htm
+image: /images/raphael-hotel.png
+tags:
+- Travel
+- Conversions
+- Engagement
+- Service Worker
+---
+
+**The Raphael Hotels'** new **PWA** increased website converstions by **20%**, increased pageviews by **66%**, sessions by **59%**, and reduced bounce rate **51%**.
```
Comment (tylersticka) [inline _posts/2017-07-13-raphael-hotel.md]: We tend not to bold the acronym "PWA" in other posts.

File: _posts/2017-07-13-raphael-hotel.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Raphael Hotel
+sourceURL: http://www.prweb.com/releases/2017/07/prweb14501233.htm
+image: /images/raphael-hotel.png
+tags:
+- Travel
+- Conversions
+- Engagement
+- Service Worker
```
Comment (tylersticka) [inline _posts/2017-07-13-raphael-hotel.md]: Nit: It's not a requirement, but we usually keep tags alphabetized.

File: _posts/2017-07-13-raphael-hotel.md (hunk 1 above)
Comment (tylersticka) [inline _posts/2017-07-13-raphael-hotel.md]: Typo: "converstions"

Comment (tylersticka) [inline _posts/2017-07-13-raphael-hotel.md]: You can maybe remove the second "increased"?

**Comment by derekshirk** [comment]
Making changes now.

File: _posts/2017-07-13-raphael-hotel.md (hunk 2 above)
Comment (derekshirk) [inline _posts/2017-07-13-raphael-hotel.md]: Good to know. Updating...

File: _posts/2017-07-13-raphael-hotel.md (hunk 1 above)
Comment (derekshirk) [inline _posts/2017-07-13-raphael-hotel.md]: Typing is hard.

**Pull request merged**